runtime/testy/studentCode.py

```diff
@@ -25,11 +25,38 @@ def nestedDict_setup():
 
 def nestedDict_main():
   print("CODE LOOP")
-
   response = Robot.getValue("dict1", "inner_dict1_int")
   print("Get Info:", response)
 
   response = 1
   Robot.setValue(response, "dict1", "inner_dict1_int")
   response = Robot.getValue("dict1", "inner_dict1_int")
-  print("Get Info2:", response)
\ No newline at end of file
+  print("Get Info2:", response)
+
+def createKey_setup():
+  Robot.createKey("Restarts")
+  Robot.setValue(0, "Restarts")
+  pass
+
+def createKey_main():
+  Robot.createKey("Restarts")
+  if Robot.getValue("Restarts") == 0:
+    try:
+      print("Making sure setValue can't create new key")
+      Robot.setValue(707, "Klefki")
+    except StudentAPIKeyError:
+      print("Success!")
+    else:
+      print("ERROR: setValue can create keys :(")
+
+  print("Creating key 'Klefki' and setting to value 707")
+  Robot.createKey("Klefki")
+  Robot.setValue(707, "Klefki")
+  print("Success!")
+
```

Comment:
You never actually check using `getValue` that you successfully set it. 
Might be good to include that just in case.